typespec-tests/tsp/enum.tsp

```diff
@@ -45,6 +45,9 @@ model Operation {
   age: 50;
   priority: Priority;
   color: ColorModel;
+  priorityValue: Priority.Low;
+  colorValue: Color.Green;
+  colorModelValue: ColorModel.Blue;
```

Comment:
The cause of the issue is this `type: EnumMember`, instead of e.g. the usual `type: Enum`